internal/controller/rabbitmq/transporturl_controller.go

```diff
@@ -304,7 +304,16 @@ func (r *TransportURLReconciler) reconcileNormal(ctx context.Context, instance *
 				Name:      userRef,
 				Namespace: instance.Namespace,
 			},
-			Spec: rabbitmqv1.RabbitMQUserSpec{RabbitmqClusterName: instance.Spec.RabbitmqClusterName, Username: instance.Spec.Username, VhostRef: vhostRef},
+			Spec: rabbitmqv1.RabbitMQUserSpec{
+				RabbitmqClusterName: instance.Spec.RabbitmqClusterName,
+				Username:            instance.Spec.Username,
+				VhostRef:            vhostRef,
+				Permissions: rabbitmqv1.RabbitMQUserPermissions{
+					Configure: ".*",
+					Read:      ".*",
+					Write:     ".*",
+				},
+			},
```

Comment:
is this required? I think it should be enough to just initialize the spec and to the assignment of the spec inside the CreateOrUpdate.